### PR TITLE
[quant] Update qbatch_norm name to qbatch_norm2d

### PR DESCRIPTION
**Pull request by supriyar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36552 [quant][graph] Add quantized batch_norm2d_relu to graph mode
* #36549 [quant][graph] Add quantized batch_norm2d support to graph mode
* #36548 [quant][graph] Add valueQuantizable function
* **#36494 [quant] Update qbatch_norm name to qbatch_norm2d**

Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21008831](https://our.internmc.facebook.com/intern/diff/D21008831)

### CODE REVIEW

File: aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp [hunk 1]
```diff
@@ -274,7 +274,7 @@ class QBatchNorm3d final : public torch::OperatorKernel {
 };
 
 static auto registry = torch::RegisterOperators().op(
-    "quantized::batch_norm(Tensor qx, "
+    "quantized::batch_norm2d(Tensor qx, "
```
Comment (jerryzh168) [inline aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp:277]: do you need to keep `quantized::batch_norm` for BC?

**Review by jerryzh168 — APPROVED** [review]

File: aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp (hunk 1 above)
Comment (supriyar) [reply aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp:277]: I don't think so. The `aten::quantized_batch_norm` is still preserving bc.
cc @lly-zero-one for new op signature.

Comment (lly-zero-one) [reply aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp:277]: Can you also change the convert script?

Comment (lly-zero-one) [reply aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp:277]: NVM, I think I just need to update the model.

**Pull request closed without merging**